.travis.yml

```diff
@@ -68,7 +69,7 @@ before_install:
 - popd
 
 script:
-- ${SPREAD_PATH}/spread -v $SYSTEM:...:$VARIANT
+- ${SPREAD_PATH}/spread -v $SYSTEM:spread/build/$TASK:$VARIANT
```

Comment:
This split looks artificial now, we could just have a single variable, explicitly listing the tasks in the `matrix` above?

Note there's more `SYSTEM` variables below, in the `sanitize` stage.